### PR TITLE
redefined-outer-name warnings

### PR DESCRIPTION
**Pull request by jenningsje**

### CODE REVIEW

File: .github/linters/.python-lint [hunk 1]
```diff
@@ -168,11 +168,10 @@ disable=abstract-method,
       range-builtin-not-iterating,
       raw_input-builtin,
       raw-checker-failed,
-      rdiv-method,
       redefined-argument-from-local,
       redefined-builtin,
       redefined-outer-name,
```
Comment (BoPeng) [inline .github/linters/.python-lint:173]: this option is not removed.

File: .github/linters/.python-lint [hunk 2]
```diff
@@ -168,11 +168,10 @@ disable=abstract-method,
       range-builtin-not-iterating,
       raw_input-builtin,
       raw-checker-failed,
-      rdiv-method,
```
Comment (BoPeng) [inline .github/linters/.python-lint:171]: and is this the one that you actually working on?

Comment (jenningsje) [reply .github/linters/.python-lint:171]: I'm not working on that one. I was working on the warning redefined-outer-name.

Comment (jenningsje) [reply .github/linters/.python-lint:171]: I'll start working on that warning right now.

**Pull request closed without merging**